src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyQueueCapacityAction.java

```diff
@@ -56,7 +56,7 @@ public String name() {
   }
 
   @Override
-  public boolean isActionable() {
+  public boolean isValid() {
```

Comment:
I liked the previous one better too, the problem was isActionable() makes sense for an action and in this case I wanted to enforce the muted-ness check in the isActionable() so that isActionable() can return false if the action is muted. The only way to do that is to override the method in an abstract class and make the method final and provide a hook for the action implementations to tell if they can perform some updates to the system(what they were doing in the original isActionable() method).

Changed it to canUpdate().